webgrid/static/webgrid.js

```diff
@@ -105,7 +105,9 @@ function datagrid_add_filter() {
 function datagrid_prep_filters(){
     $('.datagrid .filters tr').each(function(){
         jq_tr = $(this);
-        filter_key = jq_tr.attr('class');
+        // Added _filter to address CSS collision with Bootstrap
+        // Ref: https://github.com/level12/webgrid/issues/28
+        filter_key = jq_tr.attr('class').replace('_filter','');
```

Comment:
NIT: I think a regular expression would be better here so we can avoid potential problems with using the substring `_filter` in our column labels.